ambry-rest/src/main/java/com.github.ambry.rest/Http2ClientChannelInitializer.java

```diff
@@ -26,8 +26,7 @@
 
 
 /**
- * A {@link ChannelInitializer} to be used with {@link Http2BlockingChannel}. Calling {@link #initChannel(SocketChannel)}
- * adds the necessary handlers to a channel's pipeline so that it may handle requests.
+ * A {@link ChannelInitializer} to be used with {@link Http2ChannelPoolMap}.
```

Comment:
looks like this is not used with `Http2ChannelPoolMap`

Reply:
fixed. For `Http2BlockingChannel`

Reply:
This class is removed.